pages/pipelines/hosted_agents/code_access.md

```diff
@@ -1,12 +1,10 @@
 # Hosted agents code access
 
-This page details changes that need to be made to the GitHub integration to provide code access to the Buildkite hosted agents.
+Natively, Buildkite hosted agents can only be used with GitHub integrated repositories. To utilize a private repository from another provider, we've [made a plugin](https://github.com/buildkite-plugins/git-ssh-checkout-buildkite-plugin) available that's documented below.
```

Comment:
```suggestion
Natively, Buildkite hosted agents can only be used with GitHub integrated repositories. To utilize a private repository from another provider, we have [made a plugin](https://github.com/buildkite-plugins/git-ssh-checkout-buildkite-plugin) available that is documented below.
```

Reply:
total not-blocking nit 😂 not sure if we have guidelines on apostrophes, but during my uni degree I had them drilled out of me for reports / docs

Reply:
Mind you, I spoke lazy Kiwi slang and a bunch of my lecturers were German that knew English better than I did hahahhahahah